crates/server/src/admin/handlers/blob.rs

```diff
@@ -6,7 +6,10 @@ use axum::http::{header, StatusCode};
 use axum::response::{IntoResponse, Response};
 use axum::Extension;
 use calimero_primitives::blobs::BlobId;
+use calimero_store::key::BlobMeta as BlobMetaKey;
```

Comment:
```suggestion
use calimero_store::key;
```

use `key::BlobMeta` directly where you need